### PR TITLE
fix: “Copy from disruption summary” link still visible after typing into field and after clicking on it

### PR DESCRIPTION
**Pull request by alainajaffer**

### CODE REVIEW

File: site/pages/create-social-media-post/[disruptionId]/[socialMediaPostIndex].page.tsx [hunk 1]
```diff
@@ -75,7 +75,10 @@ const CreateSocialMediaPost = (props: CreateSocialMediaPostPageProps): ReactElem
                                 stateUpdater(props.disruptionSummary, "messageContent");
                             }}
                         >
-                            <p className="text-govBlue govuk-body-m">Copy from disruption summary</p>
+                            {!pageState.inputs.messageContent ||
+                            (pageState.inputs && pageState.inputs.messageContent.length === 0) ? (
+                                <p className="text-govBlue govuk-body-m">Copy from disruption summary</p>
```
Comment (alvak1) [inline site/pages/create-social-media-post/[disruptionId]/[socialMediaPostIndex].page.tsx:80]: If the message content has an error and then this button is clicked - the summary is displayed but the error is not removed

Comment (alainajaffer) [reply site/pages/create-social-media-post/[disruptionId]/[socialMediaPostIndex].page.tsx:79]: cannot be fixed due to onblur error

Comment (alainajaffer) [reply site/pages/create-social-media-post/[disruptionId]/[socialMediaPostIndex].page.tsx:79]: done

Comment (alainajaffer) [reply site/pages/create-social-media-post/[disruptionId]/[socialMediaPostIndex].page.tsx:79]: adjusted based on discussion

**Review by alvak1 — APPROVED** [review]
Approved

**Pull request merged**